### PR TITLE
cal video: show meeting info in a hideable box

### LINKED ISSUE #7293: [CAL-1139] cal video: show meeting info in a hideable box
maybe like this?

https://user-images.githubusercontent.com/8019099/220860604-576ff684-c73a-4313-a810-a810cc7b9f61.MOV

<sub>[CAL-1139](https://linear.app/calcom/issue/CAL-1139/cal-video-show-meeting-info-in-a-hideable-box)</sub>

### PR DESCRIPTION
**Pull request by PeerRich**
- [x] frontend
- [x] fixed "no meeting found" and "meeting not started" pages (had old layout) 
- [x] mobile
- [x] get data from booking data

https://user-images.githubusercontent.com/8019099/220881656-e153223a-f48f-4fe0-a8ba-9f7fc2613e8c.mp4

### CODE REVIEW

File: apps/web/pages/video/no-meeting-found.tsx [hunk 1]
```diff
@@ -1,50 +1,30 @@
...
+              {t("go_back_home")}
+            </Button>
+          }
+        />
       </main>
-    </div>
+      {/* TODO: for testing purposes only. Remove before merging */}
+      <VideoMeetingInfo />
+      <iframe src="https://cal.video/hJ3fD81SfXT8wL3nEu0N" className="h-[100vh] w-full" />
+    </>
```
Comment (PeerRich) [inline apps/web/pages/video/no-meeting-found.tsx:28]: FYI: remove this before merging. its easier to test on localhost:3000/video/no-meeting-found -- doesnt need Cal Video to be configured

**Comment by PeerRich** [comment]
great work! whats the best way we can test this?

File: apps/web/pages/video/no-meeting-found.tsx [hunk 2]
```diff
@@ -1,50 +1,30 @@
...
+            <Button data-testid="return-home" href="/event-types" EndIcon={FiArrowRight}>
+              {t("go_back_home")}
+            </Button>
+          }
+        />
       </main>
-    </div>
+      {/* TODO: for testing purposes only. Remove before merging */}
+      <VideoMeetingInfo />
+      <iframe src="https://cal.video" className="h-[100vh] w-full" />
```
Comment (PeerRich) [inline apps/web/pages/video/no-meeting-found.tsx:27]:
```suggestion
      
```

**Comment by PeerRich** [comment]
CleanShot 2023-02-24 at 11 47 26@2x
no clue why checks are failing https://vercel.com/cal/cal/5dngmkvmSeVCjckbYgtCmXXb2C4b

maybe @zomars knows more

File: apps/web/pages/video/[uid].tsx [hunk 3]
```diff
@@ -77,6 +82,112 @@ export default function JoinCall(props: JoinCallPageProps) {
...
+  const isPast = currentTime.isAfter(startingTime);
+  const currentDifference = dayjs().diff(startingTime, "minutes");
+  const startDuration = dayjs(endTime).diff(startingTime, "minutes");
+  const [duration] = useState(() => {
+    if (currentDifference >= 0 && isPast) {
+      return startDuration - currentDifference;
+    } else {
+      return startDuration;
+    }
+  });
```
Comment (Udit-takkar) [inline apps/web/pages/video/[uid].tsx:108]: why are you using useState here ?

**Review by Udit-takkar — COMMENTED** [review]
Screenshot 2023-03-04 at 2 10 22 AM

I am seeing this white boundary in the drawer. (using Brave Browser)

**Comment by PeerRich** [comment]
> Screenshot 2023-03-04 at 2 10 22 AM
> 
> 
> 
> I am seeing this white boundary in the drawer. (using Brave Browser)
> 
> 

will fix

**Comment by Udit-takkar** [comment]
@PeerRich now i don't see the arrow to open the hidde drawer

**Comment by PeerRich** [comment]
> @PeerRich now i don't see the arrow to open the hidden drawer

ahh of course... overflow-hidden...

**Review by zomars — CHANGES_REQUESTED** [review]
Blocking to apply some fixes. Neither the side bar or the button are Visible (tested on Safari and Firefox)

image

**Review by zomars — APPROVED** [review]
Tested locally on Safari/Firefox/Chrome. Waiting for preview to merge.

LGTM

**Comment by PeerRich** [comment]
epic work!

**Pull request merged**